src/main/java/org/sagebionetworks/web/client/widget/entity/PromptForValuesModalView.java

```diff
@@ -6,12 +6,69 @@
 
 import com.google.gwt.user.client.ui.IsWidget;
 
+
 /**
  * A simple model dialog prompting for multiple string values
  *
  */
 public interface PromptForValuesModalView extends IsWidget {
 
+	interface Configuration {
```

Comment:
Added a Configuration and a Builder for this modal instead of adding another `configure` method and rewiring each.

Reply:
But also added a new configureAndShow() to support the new bodyCopy param (in addition to the builder config option)?

Reply:
Can be removed, I added it before deciding to use a Builder and forgot to remove it.